setup.py

```diff
@@ -69,7 +70,7 @@ def run_tests(self):
     packages=find_packages(exclude=['barbeque.tests']),
     test_suite='.',
     tests_require=tests_require,
-    install_requires=['Django>=1.6,<1.9'],
+    install_requires=['Django>=1.6,<1.10'],
```

Comment:
I think we should update the install_requires to as we dropped Django 1.6 support.